google_analytics/aggregation.py

```diff
@@ -5,12 +5,14 @@
 
 
 def load_train_test_dataframes(data_dir, train_file_name='preprocessed_train.csv',
-                               test_file_name='preprocessed_test.csv'):
+                               test_file_name='preprocessed_test.csv', nrows_train=None, nrows_test=None):
```

Comment:
Would it be useful to include skiprows as well, so that you can process the data in multiple subsets?

Would probably bring some other complications though.

Reply:
It would but I merged before seeing this line (best maintainer ever?)